htlcswitch/link_test.go

```diff
@@ -210,6 +245,10 @@ func TestChannelLinkBidirectionalOneHopPayments(t *testing.T) {
 	if bobBandwidthBefore != n.firstBobChannelLink.Bandwidth() {
 		t.Fatal("bob bandwidth shouldn't have changed")
 	}
+
```

Comment:
Good question! This log statements were added intentionally in order to notice payment metric downgrade in case of changes inside the `htlcswitch` package, for example as have been observed by me yesterday changes in htlc pending queue (`queue.go`), might accidentally increase the `max waiting time` metric up to 2-3 times.